### PR TITLE
Add documentation for Litter-Robot integration

### PR DESCRIPTION
**Pull request by natekspencer**
## Proposed change

Add documentation for new Litter-Robot core integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/45886
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2228
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/litterrobot.markdown [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+## Configuration
+
+This integration can be configured via the UI:
+
+1. Go to **Configuration**->**Integrations**
+2. Click **+ ADD INTEGRATION** to setup a new integration
+3. Search for **Litter-Robot** and click on it
+4. Enter your _username_ and _password_ and click **SUBMIT**.
+5. Upon successful login, you will have the opportunity to select the area that your Litter-Robot is located.
+6. Click the **Finish** button.
```
Comment (frenck) [inline source/_integrations/litterrobot.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

File: source/_integrations/litterrobot.markdown [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+Replace `<entity_id>` in any of the below snippets with the appropriate value of your Litter-Robot vacuum entity.
+
+### vacuum.turn_off
+
+Supports turning off your Litter-Robot. If the unit is currently cycling, it will interrupt the cycle and stop the bonnet where it is at the time the command is received.
+
+```yaml
+service: vacuum.turn_off
+entity_id: <entity_id>
+```
```
Comment (frenck) [inline source/_integrations/litterrobot.markdown]: Please do not document services that are inherited from the vacuum platform. This comment applies to all services listed below as well.

File: source/_integrations/litterrobot.markdown [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+```yaml
+alias: Reset Waste Drawer
+sequence:
+  - service: vacuum.send_command
+    data:
+      entity_id: <entity_id>
+      command: reset_waste_drawer
+mode: single
+icon: 'mdi:refresh'
```
Comment (frenck) [inline source/_integrations/litterrobot.markdown]: This example is incomplete and its unclear what it is.
Also, make sure it follows your YAML style guide:

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

File: source/_integrations/litterrobot.markdown [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+```yaml
+service: vacuum.send_command
+data:
+  entity_id: <entity_id>
+  command: set_sleep_mode
+  params:
+    enabled: true
+    sleep_time: '22:30:00'
+```
```
Comment (frenck) [inline source/_integrations/litterrobot.markdown]: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

File: source/_integrations/litterrobot.markdown [hunk 5]
```diff
@@ -0,0 +1,111 @@
+---
+title: Litter-Robot
+description: Instructions on how to integrate a Litter-Robot WiFi-enabled, automatic, self-cleaning litter box to Home Assistant.
```
Comment (frenck) [inline source/_integrations/litterrobot.markdown]: We use the `Wi-Fi` spelling.

<https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/wi-fi>

File: source/_integrations/litterrobot.markdown [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+ha_iot_class: Cloud Polling
+ha_release: 2021.3
+ha_config_flow: true
+ha_quality_scale: gold
+ha_codeowners:
+  - '@natekspencer'
+ha_domain: litterrobot
+---
+
+The Litter-Robot integration allows you to control and monitor your WiFi-enabled, automatic, self-cleaning litter box for cats.
```
Comment (frenck) [inline source/_integrations/litterrobot.markdown]: We use the `Wi-Fi` spelling.

<https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/wi-fi>

File: source/_integrations/litterrobot.markdown [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+You will need a Litter-Robot account as well as a Wi-Fi enabled Litter-Robot unit that has already been associated with your account.
```
Comment (frenck) [inline source/_integrations/litterrobot.markdown]:
```suggestion
You will need a Litter-Robot account as well as a Wi-Fi-enabled Litter-Robot unit that has already been associated with your account.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/litterrobot.markdown [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+
+### reset_waste_drawer
+
+Resets the waste drawer gauge on the Litter-Robot. This will reset the cycle count returned by the Litter-Robot API to `0`.
+
+```yaml
+service: vacuum.send_command
+data:
+  entity_id: vacuum.litter_robot_litter_box
+  command: reset_waste_drawer
```
Comment (frenck) [inline source/_integrations/litterrobot.markdown]:
```suggestion
service: vacuum.send_command
target:
  entity_id: vacuum.litter_robot_litter_box
data:
  command: reset_waste_drawer
```

File: source/_integrations/litterrobot.markdown [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+Example of setting the sleep mode to begin at 10:30 PM.
+
+```yaml
+service: vacuum.send_command
+data:
+  entity_id: vacuum.litter_robot_litter_box
+  command: set_sleep_mode
+  params:
+    enabled: true
+    sleep_time: "22:30:00"
```
Comment (frenck) [inline source/_integrations/litterrobot.markdown]:
```suggestion
service: vacuum.send_command
target:
  entity_id: vacuum.litter_robot_litter_box
data:
  command: set_sleep_mode
  params:
    enabled: true
    sleep_time: "22:30:00"
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by natekspencer** [comment]
@frenck hopefully those are all the updates needed now!

**Review by frenck — APPROVED** [review]
Thanks, @natekspencer 👍

**Pull request merged**